pom.xml

```diff
@@ -167,13 +167,34 @@
             </goals>
           </execution>
         </executions>
+        <dependencies>
+          <dependency>
+            <groupId>net.sf.proguard</groupId>
+            <artifactId>proguard-base</artifactId>
+            <version>5.2.1</version>
+            <scope>runtime</scope>
+          </dependency>
+        </dependencies>
         <configuration>
+          <proguardVersion>5.2.1</proguardVersion>
           <obfuscate>false</obfuscate>
           <options>
             <option>-allowaccessmodification</option>
             <option>-keep class com.squareup.haha.perflib.** { *; }</option>
+            <option>-keep class com.squareup.haha.trove.THashMap { *; }</option>
```

Comment:
See PR comment above. Updating Proguard causes the jar size to drop significantly. I have to assume some unused classes were being included in the jar and Proguard improvements changed that. THashMap is being used in Leak Canary.